packages/cli/mcp/tools.ts

```diff
@@ -0,0 +1,266 @@
+  }
+
+  const stages = await listStages(appId);
+
+  // Add features tool
+  mcp.tool(
+    "features",
+    "List all feature flags of the Bucket feature management service.",
+    async () => {
+      try {
+        const data = await listFeatureNames(appId);
+        return {
+          content: [
+            {
+              type: "text",
+              text: `
+List of features.
+>>> JSON Response >>>
+ ${JSON.stringify(data, null, 2)}`,
+            },
+          ],
+        };
+      } catch (error) {
+        return await handleMcpError(error);
+      }
+    },
+  );
+
+  // Add feedback tool
+  mcp.tool(
+    "feedback",
+    "Get user feedback for the features of the Bucket feature management service.",
+    withDefaults(FeedbackQuerySchema, {
+      envId: production.id,
+    }).shape,
+    async (args) => {
+      try {
+        const data = await listFeedback(appId, args);
+        return {
+          content: [
+            {
+              type: "text",
+              text: `
+Feedback is returned in a JSON format with pages.
+Feedback score is between 1 and 5, with 0 being unknown.
+>>> JSON Response >>>
+${JSON.stringify(data, null, 2)}`,
+            },
+          ],
+        };
+      } catch (error) {
+        return await handleMcpError(error);
+      }
+    },
+  );
+
+  // Add create feature tool
+  const keyFormatRules = KeyFormatPatterns[org.featureKeyFormat].message;
+  mcp.tool(
+    "featureCreate",
+    "Creates a new feature flag of the Bucket feature management service.",
+    withDescriptions(FeatureCreateSchema, {
+      key: `Feature key specified in ${org.featureKeyFormat} format:\n${keyFormatRules}`,
+    }).shape,
+    async (args) => {
+      try {
+        const feature = await createFeature(appId, args);
+
+        const featureLink = featureUrl(
+          configStore.getConfig("baseUrl"),
+          production,
+          feature,
+        );
+        return {
+          content: [
+            {
+              type: "text",
+              text: `Feature created successfully. See it [on Bucket here](${featureLink}). Before moving on we need to generate the refreshed types to avoid linting errors. Run the Bucket CLI in a terminal:
+\`shell
+$ npx bucket features types
+\`
+
+After that we can feature flag some code. Use the following pattern for React:
+
+\`\`\`typescript
+import { useFeature } from "@bucketco/react-sdk";
+function MyComponent() {
+  const { isEnabled } = useFeatureFlag("${feature.key}");
+  if (!isEnabled) {
+    // feature is disabled
+    return null;
+  }
+  return <div>Feature is disabled.</div>;
+}
+\`\`\`
+
+To track feature usage, call the \`track\` method on usage:
+
+\`\`\`typescript
+import { useFeature } from "@bucketco/react-sdk";
+
+function MyComponent() {
+  const { isEnabled } = useFeatureFlag("${feature.key}");
+  if (!isEnabled) {
+    // feature is disabled
+    return null;
+  }
+  return <button onClick={track}>Start my feature!.</button>;
+}
```

Comment:
@roncohen @Swiftwork FYI, this code needs to use the `track` (i assume you just forgot to update it when copy/pasting above)